### PR TITLE
Credit more contributors in AUTHORS

### PR DESCRIPTION
**Pull request by AA-Turner**
This should mean ~all contributors with more than ~20 commits are listed in AUTHORS.

I've made a best-effort listing of everyone's main area of contribution, but if you would like to list something else and/or reword, please let me know!

cc: 
* @Blendify
* @adamtheturtle 
* @alex 
* @brechtm 
* @choldgraf 
* @danieleades 
* @blueyed
* @DimitriPapadopoulos 
* @mitya57 
* @larsoner 
* @eric-wieser 
* @jbms 
* @jdufresne 
* @astrojuanlu 
* @keewis 
* @lmmx 
* @marxin 
* @mgeier 
* @septatrix 
* @rffontenelle 

xref:
* #13080
* #13090 

A

### CODE REVIEW

**Review by astrojuanlu — APPROVED** [review]
❤️

**Comment by adamtheturtle** [comment]
♥️

**Review by septatrix — APPROVED** [review]

**Comment by larsoner** [comment]
@AA-Turner FYI over in MNE-Python we use a monthly action for code credit:

https://github.com/mne-tools/mne-python/blob/main/.github/workflows/credit.yml

You could do something similar here I think. The bot would open a PR with a (hopefully!) correct name from the `git log` plus some placeholder text that you could then edit on the GitHub interface.

Happy to give it a shot if you think it would make maintenance easier for you!

File: AUTHORS.rst [hunk 1]
```diff
@@ -53,25 +64,34 @@ Contributors
...
 * Kevin Dunn -- MathJax extension
 * KINEBUCHI Tomohiko -- typing Sphinx as well as docutils
 * Kurt McKee -- documentation updates
 * Lars Hupfeldt Nielsen - OpenSSL FIPS mode md5 bug fix
+* Louis Maddox -- better docstrings
 * Łukasz Langa -- partial support for autodoc
 * Marco Buttu -- doctest extension (pyversion option)
 * Martin Hans -- autodoc improvements
 * Martin Larralde -- additional napoleon admonitions
+* Martin Liška -- option directive and role improvem`ents
```
Comment (rffontenelle) [inline AUTHORS.rst]: Typo in improvements

**Pull request merged**